conf/nginx.conf

```diff
@@ -12,11 +12,6 @@ location __PATH__/ {
   # Path to source
```

Comment:
According to [this doc](https://sabre.io/baikal/upgrade/):

> If it was before something like:
> 
> http://example.org/baikal/cal.php
>
> Then it's now:
>
> http://example.org/baikal/html/dav.php
>
> So two things changed:
>
>    cal.php, card.php moved into the html/ directory.
>    We added dav.php. cal.php and card.php still exist in the html/ directory but will be removed in a future version. > dav.php is an endpoint for both caldav and carddav combined.

With our nginx conf we can still remove `html` from the url, but we must change
```nginx
location = /.well-known/carddav {
  return 301 https://$server_name__PATH__/card.php;
}
location = /.well-known/caldav {
  return 301 https://$server_name__PATH__/cal.php;
}
```

to:
```nginx
location = /.well-known/carddav {
  return 301 https://$server_name__PATH__/dav.php;
}
location = /.well-known/caldav {
  return 301 https://$server_name__PATH__/dav.php;
}
```